README.md

```diff
@@ -1,33 +1,41 @@
 Tax Calculator
-=======
-The Tax Calculator is a calculator for the US federal individual income tax system. In conjunction with micro data that represents US taxpayer characteristics and a set of behavioral assumptions, we use the Tax Calculator to conduct revenue estimation and distributional analyses of tax policies. 
+==============
 
-Documentation for the Tax calculator is available at http://taxcalc.readthedocs.org/en/latest/.
+The Tax Calculator simulates the US federal individual income tax
+system.  In conjunction with micro data that represent the US
+population and a set of behavioral assumptions, the Tax Calculator can
+be used to conduct revenue scoring and distributional analyses of tax
+policies.  The Tax Calculator is written in Python, an interpreted
+language that can execute on Windows, Mac, or Linux.
 
 About OSPC
-=======
-The Open-Source Policy Center (OSPC) seeks to make policy analysis more transparent, trustworthy, and collaborative by harnessing open-source methods to build cutting-edge economic models. 
-
-Disclaimer
-========
-The Tax-Calculator is currently under development. Therefore, there is NO GUARANTEE OF ACCURACY. THE CODE SHOULD NOT CURRENTLY BE USED FOR PUBLICATIONS, JOURNAL ARTICLES, OR RESEARCH PURPOSES. DO NOT CITE. Furthermore, users should be forewarned that the interface could change significantly, and the implementation is subject to wild change. The code is currently available for testing purposes only.
+==========
 
+The Open Source Policy Center (OSPC) seeks to make policy analysis
+more transparent, trustworthy, and collaborative by harnessing
+open-source methods to build cutting-edge economic models.
 
-Installation
```

Comment:
On Sat, Jun 20, 2015 at 9:55 AM, T.J. Alumbaugh <notifications@github.com>
wrote:

> In README.md
> <https://github.com/OpenSourcePolicyCenter/Tax-Calculator/pull/289#discussion_r32884665>
> :
>
> >
> > -Installation
>
> Why are we removing the part that tells people how to install the package?
> That seems like valuable information.
>
T.J.,

Why?  Because the current Installation instructions are confusing.  See the
following email sent to Matt Jensen about my experience in trying to follow
the existing Installation instructions.

Matt,

While I am waiting for reactions to my pull request, I decided to
look over the Installation section of the README.md file, which
is the first thing somebody who is interested in getting started
with the Tax Calculator will read.

My overall reaction is that the README.md Installation
instructions are not well designed to get people started and, in
fact, are likely to discourage people from getting started
with the Tax Calculator.  My experience is based on creating
a new Mac user and trying to follow the Installation instructions.
My experience was a bit like opening *War and Peace* in the
middle of the book and starting to read: pretty confusing.

I would have tried to prepare a pull request for modifications
of README.md, but have no idea about how to approach
revisions.  Here are some notes I made while trying to follow
the instructions:

(a) In the first section beginning with "The taxcalc package is
installable via" the "python setup.py install" command does nothing
even if python is installed (as, say, on a Mac).  Where is the setup.py
file supposed to be located on the local computer?  Or, if it to be
downloaded, how does an interested party do that?

(b) Even if the Anaconda distribution of Python is installed
(which the Installation section says nothing about), the command
"conda install -c ospc taxcalc" does work, but there are two major
problems:

First, this command's output says NOTHING about where taxcalc
is installed.  After much looking, I found it here:
~/anaconda/lib/python3.4/site-packages/taxcalc-0.4.9-py3.4.egg/taxcalc
It is bad form to install an application (the Tax Calculator) in the
same directory tree as the Anaconda tools; certainly the instructions
in the contributor_guide.rst do not call for such an approach.

And the second problem, is that when the user changes to that directory,
and executes "py.test" (which is something that the documentation,
rightly so, urges people to do) nothing happens except the message
"collected 0 items".  This is probably because there is no tests
subdirectory in the taxcalc directory (as there is when following the
installation instructions in the Contributor Guide).

(c) The final section begins with "For contributors, the conda recipe
is located in the conda.recipe directory."  But the installation in
step (b) above does not include a conda.recipe file or directory.
So, again, there is nothing but confusion for somebody trying to
get started with the OSPC Tax Calculator.

I'd be happy to prepare some revisions to README.md, but we
would need to talk some about what you want it to say.

Best wishes, Martin

Matt replied to that email as follows:

Hi Martin,

I wholeheartedly agree that the installation section of README.md is
confusing.

What would you think of something along the lines of the snippet below
(perhaps written or laid out differently, but expressing this content?)

"The taxcalc package is currently available to contributors. To get
started, please read our contributor guide <
http://taxcalc.readthedocs.org/en/latest/contributor_guide.html>.

If you would like to access the package through our web application, please
email matt.jensen@aei.org about becoming a beta tester for our TaxBrain web
application at ospc.org."

This kicks the can down the road a bit, but I’m not sure we are ready to
interact with the type of user who only wants to use the taxcalc package
api without digging into the source; and if someone does want to dig into
the source, I’d like to encourage them to be set up with git and
contribute.

Happy to talk about this on the phone. 202.862.5941

Best, Matt

So, pull request #289 is my effort to implement Matt's suggestions.

Do you have some concrete suggestions you would like to be
considered for inclusion in a revised pull request?

Best wishes,  Martin